test/tx-builder/dydx-governance.spec.ts

```diff
@@ -53,6 +58,27 @@ describeContract('DydxGovernance', init, (ctx: TestContext) => {
         );
         expect(emptyVoters.size).to.be.eq(0);
       });
+
+      it('getUserBalancesPerEpoch', async () => {
+        const userBalancesPerEpoch = await txBuilder.merkleDistributorService.getUserBalancesPerEpoch();
+        expect(Object.keys(userBalancesPerEpoch).length).to.be.eq(3);
+        expect(Object.keys(userBalancesPerEpoch['2']).length).to.be.eq(40867);
+      });
+
+      it('getActiveUsersInEpoch 0', async () => {
+        const activeUsers = await txBuilder.merkleDistributorService.getActiveUsersInEpoch(0);
+        expect(activeUsers.length).to.be.eq(32457);
+      });
+
+      it('getActiveUsersInEpoch 1', async () => {
+        const activeUsers = await txBuilder.merkleDistributorService.getActiveUsersInEpoch(1);
+        expect(activeUsers.length).to.be.eq(9979);
+      });
+
+      it('getActiveUsersInEpoch 11', async () => {
```

Comment:
Nit: note that this is a future epoch in the test name, also use an epoch way in the future so this test doesn't fail for a while (maybe epoch 500?).